docs/components/console/manage-clusters/cluster-capacity.md

```diff
@@ -0,0 +1,104 @@
+
+:::info
+To understand how cluster capacity is calculated, see [how cluster capacity is calculated](#capacity-calculation).
+:::
+
+## View cluster capacity
+
+The cluster capacity percentage is shown in the **Capacity** column on the Console **Clusters** tab.
+
+<img src={CapacityPercentImg} alt="Example cluster capacity percentage" />
+
+Select a cluster to view detailed cluster capacity information on the cluster **Overview** tab.
+
+<img src={CapacityImg} alt="Example cluster capacity" style={{width: '550px'}}/>
+
+- The current cluster capacity is shown as a percentage bar at the top of the section.
+- The chart shows detailed capacity data for the last 24 hours, 7 days, or 30 days.
+  - Select the time period you want to view data for.
+  - Hover over individual nodes in the chart to view data for a specific time or day.
+- Click the **Refresh** icon to refresh and update the cluster capacity data.
+
+## Manage cluster capacity
+
+Cluster capacity will fluctuate based on incoming user requests, and internal processing load.
+
+- User requests are those sent directly by an external client.
+- Internal load refers to all other processing that is **not** directly triggered by a client. For example, timer events, a job being made available after back off, and so on.
+
+This means that cluster capacity could fluctuate throughout the day. For example:
+
+- Capacity might be higher during business hours, but lower overnight when the cluster is unused.
+- Certain processes can cause a "fan-out" effect, where even though creating a process instance is only a single user request, it could require computation on a very large multi-instance collection, resulting in a high capacity spike.
+- You could have a spike in capacity if a large number of timers are triggered at the same time.
+
+### High capacity
+
+A high capacity does not necessarily mean that a cluster needs to be resized, but it could indicate that your cluster is overloaded.
+
+> A high capacity benchmark is relative, but generally **anything above 60%** is considered high capacity.
+
+If the capacity is high, but your operations are completing within an acceptable timeframe, requests are successful, and so on, then you do not need to take any action.
+
+If you find yourself experiencing any of the following, then you might need to investigate and take action:
+
+- REST clients are receiving a high number of `429` errors.
+- gRPC clients are receiving a high number of `RESOURCE_EXHAUSTED` errors.
+- More and more commands or queries are timing out in your clients.
+- Web components are slowing down, or showing data that is severely out of date (for example, 1 hour).
+
+In this scenario, you should look at reducing the overall load on the cluster to try and lower the cluster capacity value.
+
+### Reduce cluster capacity
+
+If your cluster capacity is too high, you can help reduce cluster capacity by:
```

Comment:
:wrench: Again, this would be easier to understand if we said "If your cluster utilization is too high" or "If your cluster does not have much capacity available"